CarpetLib/src/restrict_3d_stagger110.cc

```diff
@@ -145,8 +145,8 @@ void restrict_3d_stagger110(
 
   } else {
 
-// Loop over coarse region
-#pragma omp parallel
+    // Loop over coarse region
+    // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.